tests/xoto3/dynamodb/write_versioned/conftest.py

```diff
@@ -0,0 +1,48 @@
+from typing import Callable, cast
```

Comment:
Does this represent the setup we would need to unit test our own TransactionBuilder functions?

Reply:
in general i don't think even this part is absolutely necessary.

I think I will add something to the readme to demonstrate a canonical test.

Reply:
i have added a pair of example unit tests that go with the example in the README, and I think they illustrate the intended approach fairly thoroughly.